packages/mdc-switch/README.md

```diff
@@ -40,9 +40,11 @@ npm install @material/switch
 
 ```html
 <div class="mdc-switch">
-  <input type="checkbox" id="basic-switch" class="mdc-switch__native-control" role="switch">
-  <div class="mdc-switch__background">
-    <div class="mdc-switch__knob"></div>
+  <div class="mdc-switch__track"></div>
+  <div class="mdc-switch__thumb-underlay">
+    <div class="mdc-switch__thumb">
+        <input type="checkbox" id="basic-switch" class="mdc-switch__native-control" role="switch">
+    </div>
```

Comment:
The basic usage section seems to be missing the Styles (for imports) and JavaScript Instantiation sections

Reply:
Good catch, will add!